unauthorized/index.php

```diff
@@ -0,0 +1,20 @@
+<?php $root = realpath($_SERVER["DOCUMENT_ROOT"]) ?>
```

Comment:
I kind of like this idea. Another alternative is returning a HTTP 403 but I think this one is nice because we can end up using this page as a sort of readme on permissions like "please contact your site coordinator if you need access" or similar.

Reply:
Idea being the entire page.

Reply:
I also think this is nice since this isn't exactly an API, but rather like a user facing site, an unauthorized thing looks nice and feels cleaner.